janusgraph-core/src/main/java/org/janusgraph/graphdb/tinkerpop/JanusGraphFeatures.java

```diff
@@ -78,7 +78,7 @@ public static JanusGraphFeatures getFeatures(StandardJanusGraph graph, StoreFeat
 
         @Override
         public boolean supportsMapValues() {
-            return false;
+            return true;
```

Comment:
Can you speak a bit more about the test failures that required the change?  Are there TinkerPop tests that were requiring map support even with the feature disabled?  Regardless, I'm ok with moving forward on this to get it merged but I'll start a discussion on the list so that we can reach consensus on whether or not `map` support should be added back in and if it is, what the extra validation implications are.  At that point, if we move forward with it, we can create an issue.